pyriemann_qiskit/classification.py

```diff
@@ -0,0 +1,317 @@
+"""Module for classification function."""
+import numpy as np
+
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.model_selection import train_test_split
+
+from qiskit import BasicAer, IBMQ
+from qiskit.circuit.library import ZZFeatureMap, TwoLocal
+from qiskit.aqua import QuantumInstance, aqua_globals
+from qiskit.aqua.quantum_instance import logger
+from qiskit.aqua.algorithms import QSVM, SklearnSVM, VQC
+from qiskit.aqua.utils import get_feature_dimension
+from qiskit.providers.ibmq import least_busy
+from qiskit.aqua.components.optimizers import SPSA
+from datetime import datetime
+import logging
+logger.level = logging.INFO
+
+
+class QuanticBase(BaseEstimator, ClassifierMixin):
+
+    """Quantum classification.
+
+    This class implements a SKLearn wrapper around Qiskit library.
+    It provides a mean to run classification tasks on a local and
+    simulated quantum computer or a remote and real quantum computer.
+    Difference between simulated and real quantum computer will be that:
+    - There is no noise on a simulated quantum computer (so results are better)
+    - Real quantum computer are quicker than simulator
+    - Real quantum computer tasks are assigned to a queue
+      before being executed on a back-end
+
+    WARNING: At the moment this implementation only supports binary
+    classification (eg. Target vs Non-Target experiment)
+
+    Parameters
+    ----------
+    target : int
+        Label of the target symbol
+    qAccountToken : string (default:None)
+        If quantum==True and qAccountToken provided,
+        the classification task will be running on a IBM quantum backend
+    processVector : lambda vector: processedVector (default)
+        Additional processing on the input vectors. eg: downsampling
+    verbose : bool (default:True)
+        If true will output all intermediate results and logs
+    quantum : Bool (default:True)
+        - If true will run on local or remote backend
+        (depending on qAccountToken value).
+        - If false, will perform classical computing instead
+    **parameters : dictionnary
+        This is used by  SKLearn with get_params and set_params method
+        in order to create a deepcopy of an instance
+
+    Attributes
+    ----------
+    classes_ : list
+        list of classes.
+    verbose : see above
```

Comment:
`verbose` is an input parameter, it is not a inner attribute of the class.
Same remark for parameters below.

Moreover, in scikit conventions, attributes have a _ at the end of their names.

Reply:
Please, note that in the constructor: `self.verbose = verbose` (also apply for parameters below).

The naming convention was updated as requested.